doc/sphinx/source/recipes/recipe_modes_of_variability.rst

```diff
@@ -43,8 +43,9 @@ User setting files are stored in recipes/
    * end_historical: end date (YYYY-MM-DD) of the reference dataset to be used (please make sure this matches the available data)
    * start_projection: start date (YYYY-MM-DD) of the projection dataset to be used (please make sure this matches the available data)
    * end_projection: end date (YYYY-MM-DD) of the projection dataset to be used (please make sure this matches the available data)
-   * region: North-Atlantic or Polar
-   * ncenters: number of centers to be computed by the k-means clustering algorithm (does not work yet)
+   * plot type: rectangular or polar
```

Comment:
'polar' is 'Polar' in the diagnostic script, so this will not work:
https://github.com/ESMValGroup/ESMValTool/blob/1b792826d74fdbb84c39c0d785c2ec3b74082283/esmvaltool/diag_scripts/magic_bsc/weather_regime.r#L279

Reply:
corrected

Reply:
Did you upload the changes? I don't see them